assets/css/styles.css

```diff
@@ -1,3 +1,38 @@
+    html{
```

Comment:
No need of this because its already there @Devemmy01

Reply:
My bad. it's fixed